tests/playwright/sanity/container.test.js

```diff
@@ -261,4 +261,49 @@ test.describe( 'Container tests', () => {
 			container: false,
 		} );
 	} );
+
+	test( 'Widget display inside container flex wrap', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+		} );
+
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' ),
+			containerElement = editor.getPreviewFrame().locator( '.elementor-edit-mode .elementor-element-' + container );
+
+		// Set row direction.
+		await page.click( '.elementor-control-flex_direction i.eicon-arrow-right' );
+		// Set flex-wrap: wrap.
+		await page.click( '.elementor-control-flex_wrap .elementor-control-input-wrapper .eicon-wrap' );
+
+		// Close Navigator
+		await editor.closeNavigatorIfOpen();
+
+		// Act.
+		await editor.addWidget( 'google_maps', container );
+		// Set widget custom width to 40%.
+		await wpAdmin.setWidgetCustomWidth( '40' );
+		await wpAdmin.setWidgetToFlexGrow();
+		// Set widget mask.
+		await wpAdmin.setWidgetMask();
```

Comment:
@udidol I set a widgetMask, because the Video & Google Map sometimes don't render 100% in the test. 
Most of the times the Youtube and Google Maps controls are causing problems. I have tried to reduce the glitches, by taking a screenshot of the right top corner section only.

Reply:
@udidol 
On the left hand side there are more glitches (with various video and map controls)